Readme.md

```diff
@@ -0,0 +1,234 @@
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+-->
+# 📖 Catalog of My Things
+
+<div align="center">
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+    - [🚀 demo video](#demo-video)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+## 📖 Catalog of My Things <a name="about-project"></a>
+
+> This is a console app that will help you to keep a record of different types of things you own: books, music albums, movies, and games. Everything will be based on the UML class diagram presented below. The data will be stored in JSON files but there is also a database with tables structure analogical to the program's class structure.
+
+<img src = "./Capture.png">
+
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> Ruby , Rspec, Sql, Postgres
+
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://docs.rubocop.org/">Ruby</a></li>
+  </ul>
+</details>
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[keep a record of different types of things you own]**
+- **[Store the data in JSON files]**
+- **[Add items you own]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+## demo-video
+> group demo video for the presentation of the app
+
+[Link to demo video](https://youtu.be/qYq3NXRCJrU)
```

Comment:
- [ ] Good job recording a video. However, 11 minutes is way longer than the required length of the video. Please make sure that the video has a maximum length of 5 minutes. As per [this](https://github.com/microverseinc/curriculum-ruby/blob/main/group-capstone/catalog_of_my_things.md#:~:text=It%20is%20a%20video%20with%20a%20maximum%20length%20of%205%20minutes.) requirement.